subsys/net/lib/coap/Kconfig

```diff
@@ -62,6 +62,15 @@ config COAP_EXTENDED_OPTIONS_LEN_VALUE
 	COAP_EXTENDED_OPTIONS_LEN is enabled. Define the value according to
 	user requirement.
 
+config COAP_MBEDTLS_SSL_MAX_CONTENT_LEN
+	int "CoAP MBEDTLS maximum content length value"
+	default 1500
```

Comment:
Does this have any minimum size? If so, please add a `range` statement.  Otherwise, this change LGTM.